detection_rules/etc/non-ecs-schema.json

```diff
@@ -112,6 +112,9 @@
   "logs-google_workspace*": {
     "gsuite.admin": "keyword",
     "gsuite.admin.new_value": "keyword",
-    "gsuite.admin.setting.name": "keyword"
+    "gsuite.admin.setting.name": "keyword",
```

Comment:
Are these fields new for a yet to be released integration version (or beats 8.8)? Why do they need to be added here

Reply:
the example data is from 8.5. Do we just need to refresh the integrations and update beats?

Reply:
I believe these were added when we did the integration validation PR. The GSuite integration was renamed to Google Workspace after 7.16. Therefore the rules in 7.16 failed unit tests as there is no `gsuite` integration to validate against nor is it available in EPR because of the rename. As a result, we needed to add this to the `non-ecs` file and let it backport to pass validation. @Mikaayenson is this correct?

Example: https://github.com/elastic/detection-rules/blob/7.16/rules/integrations/google_workspace/google_workspace_mfa_enforcement_disabled.toml

Reply:
By the way, only the `google_workspace*` fields were added for this PR. They were not passing schema checks related to the query being EQL. I will have too dig deeper into this.

Reply:
<img width="1030" alt="Screenshot 2023-03-07 at 9 20 03 PM" src="https://user-images.githubusercontent.com/99630311/223602956-2281da36-88dc-4d03-8b04-e445d9485647.png">

@brokensound77 So it looks like it HAS to be an ECS or Beats schema field, if not validation will fail because our logic is "valid ECS/Beats  _AND_ valid integration". If it were an OR, it would not find it in ECS/Beats but find it in the integrations schema next, therefore it had to be added to `non-ecs` file.